lib/percy/client/snapshots.rb

```diff
@@ -14,6 +14,7 @@ def create_snapshot(build_id, resources, options = {})
             'attributes' => {
               'name' => options[:name],
               'enable-javascript' => options[:enable_javascript],
+              'enable-plain-rcw' => options[:disable_transformations],
```

Comment:
```suggestion
              'disable-transformations' => options[:disable_transformations],
```
Let's stick to the same format throughout the pipeline?

Reply:
Also please get a review from @itsjwala or @samarsault on this.

Reply:
Looks good, update specs and description.